config/app/init/initializers/prefilled-root.json

```diff
@@ -18,8 +18,8 @@
           "templateFolder": "@css:templates/root/prefilled",
           "factory": { "@type": "ExtensionBasedMapperFactory" },
           "templateEngine": {
-            "@type": "HandlebarsTemplateEngine",
-            "baseUrl": { "@id": "urn:solid-server:default:variable:baseUrl" }
+            "@type": "GenericTemplateEngine",
+            "handler": { "@id": "urn:solid-server:default:GenericTemplateEngine" }
```

Comment:
Would name the parameter `templateEngine` for clarity.

Would also use `urn:solid-server:default:TemplateEngine` because now it's confusing that `urn:solid-server:default:GenericTemplateEngine` is not a `GenericTemplateEngine`.